RealmSwift/Object.swift

```diff
@@ -193,6 +193,50 @@ open class Object: RLMObjectBase, ThreadConfined {
         }
     }
 
+    // MARK: Notifications
+
+    /**
+     Observe this object for changes, including those made on other threads and
```

Comment:
The wording differs between the objc and swift docs and could probably mostly be realigned.